examples/variational_autoencoder.py

```diff
@@ -40,8 +37,8 @@ def sampling(args):
 x_decoded_mean = decoder_mean(h_decoded)
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
-    kl_loss = - 0.5 * K.mean(1 + z_log_std - K.square(z_mean) - K.exp(z_log_std), axis=-1)
+    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
`x^{(i)}` is a single training example (e.g. image) in Eq. 10 of Kingma's VAE paper. 

In Eq. 11, `x_i` refers to the i-th dimension of the training example (e.g. pixel).